parm/postxconfig-NT-fv3lam_rrfs.txt

```diff
@@ -10747,6 +10747,43 @@ surface
 ?
 ?
 ?
+1006
```

Comment:
@JesseMeng-NOAA The new variable should be incorporated in one RRFS dataset only. You could revert the changes in PRSLEVE dataset.

Reply:
Done